tardis/adapters/sites/moab.py

```diff
@@ -72,12 +72,29 @@ def __init__(self, machine_type: str, site_name: str):
             remote_resource_uuid="JobID", resource_status="State"
         )
 
+        # see job state codes at https://computing.llnl.gov/tutorials/moab/#JobStates
         translator_functions = StaticMapping(
             State=lambda x, translator=StaticMapping(
+                BatchHold=ResourceStatus.Stopped,
+                Canceling=ResourceStatus.Running,
+                Completed=ResourceStatus.Deleted,
+                Deffered=ResourceStatus.Booting,
+                Depend=ResourceStatus.Error,
+                Dependency=ResourceStatus.Error,
```

Comment:
Comparing your states to what is written in https://computing.llnl.gov/tutorials/moab/#JobStates, `FAILED` and `NODE_FAIL` are missing and `SystemHold` as well as `UserHold` is not mentioned in the webpage. What is the reason for that?

Reply:
I did not add them, because I've not encountered them using `showq --xml`. However, I've added them now for completeness.